client/asset/dcr/dcr.go

```diff
@@ -157,6 +157,7 @@ type rpcClient interface {
 	GetBestBlock(ctx context.Context) (*chainhash.Hash, int64, error)
 	GetBlockHash(ctx context.Context, blockHeight int64) (*chainhash.Hash, error)
 	GetBlockVerbose(ctx context.Context, blockHash *chainhash.Hash, verboseTx bool) (*chainjson.GetBlockVerboseResult, error)
+	GetBlockHeaderVerbose(ctx context.Context, blockHash *chainhash.Hash) (*chainjson.GetBlockHeaderVerboseResult, error)
```

Comment:
Did you need the **verbose** block header? `wire.BlockHeader` has a `TimeStamp` and a `PrevBlock`.

Reply:
Hmm, did i not? In the future the median time will be included in the verbose results. I think.

Reply:
Good point @JoeGruffins.  Link to your dcrd PR so we can find it later: https://github.com/decred/dcrd/pull/2638#pullrequestreview-642063821